README.md

```diff
@@ -22,11 +22,50 @@
 $ npm i @helia/remote-pinning
 ```
 
-A longer repository description.
-
 ## Documentation
 
-[Insert link to documentation]() or expand with Install, Build, Usage sections.
+### Create remote pinner
+
+```typescript
+import { unixfs } from '@helia/unixfs'
+import { Configuration, RemotePinningServiceClient } from '@ipfs-shipyard/pinning-service-client'
+import { createHelia } from 'helia'
+import { createRemotePinner } from '../src/index.js'
```

Comment:
People will copy/paste this to get started so it's probably worth importing from the published module name:

```suggestion
import { createRemotePinner } from '@helia/remote-pinning'
```